test/tests/imageserver/test_image_service_system.py

```diff
@@ -0,0 +1,385 @@
+'''
+Copyright 2017 Dell Inc. or its subsidiaries.  All Rights Reserved.
+
+Author(s):
+Norton Luo
+This test validate the system level function of RackHD image-service. This test include VMware ESXi install and node
+rediscover.It use image server to store ESXi image and microkernel used for RackHD discovery.
+You need put an config file in the /config directory
+'''
+
+import os
+import sys
+import time
+import flogging
+import random
+import fit_common
+import urllib2
+import pexpect
+import unittest
+import test_api_utils
+from nose.plugins.attrib import attr
+logs = flogging.get_loggers()
+
+
+@attr(all=False, regression=False, smoke=False, imageservice=False)

```

Comment:
We are still working on the images ervice CI test work flow. Currently, there is no RackHD in the system. So this system test will be disabled by default.